src/test/globalSetup.test.ts

```diff
@@ -45,6 +46,7 @@ before(async function () {
     ext.init(fakeContext, extWindow.Window.vscode())
     ext.telemetry = service
     await initializeComputeRegion()
+    lazyLoadSamTemplateStrings()
```

Comment:
Nit: maybe move this out of the global setup and into the before for `samTemplates.test.ts`?

Reply:
Oh yeah I didn't even realize it was called `SamTemplateStrings` in particular